chapter1/rotateMatrix.js

```diff
@@ -0,0 +1,44 @@
+// Given an image represented by an NxN matrix, where each pixel in the image is 4 bytes, write a method to rotate the image by 90 degrees.
+
+/*
+Input:
+[
+  [1,2,3],
+  [4,5,6],
+  [7,8,9]
+];
+Output: 
+[
+  [7,4,1],
+  [8,5,2],
+  [9,6,3]
+];
+
+*/
```

Comment:
I dont see any issues with your logic so I only have suggestions for syntax.  Nitpicky stuff but a little polish goes a long way in an interview...